### PR TITLE
Add csv bulk insert test

### PR DESCRIPTION
**Pull request by KumaJie**
issue: #28188

### CODE REVIEW

File: tests/python_client/common/bulk_insert_data.py [hunk 1]
```diff
@@ -521,4 +521,90 @@ def prepare_bulk_insert_numpy_files(minio_endpoint="", bucket_name="milvus-bucke
...
+    :type data_fields: list
+
+    :param file_nums: file numbers to be generated
+    :type file_nums: int
+
+    :param force: re-generate the file(s) regardless existing or not
+    :type force: boolean
+    """
+    data_fields_c = copy.deepcopy(data_fields)
+    print(f"data_fields: {data_fields}")
```
Comment (binbinlv) [inline tests/python_client/common/bulk_insert_data.py:606]: could you please use log.info() or log.debug() to print log in our test framework?

Comment (KumaJie) [reply tests/python_client/common/bulk_insert_data.py:606]: got it.

**Pull request merged**